[2023-10-20 23:22:43,286] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-05-19T00:00:00+00:00 [queued]>
[2023-10-20 23:22:43,299] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-05-19T00:00:00+00:00 [queued]>
[2023-10-20 23:22:43,300] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:22:43,301] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-10-20 23:22:43,301] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:22:43,320] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2023-05-19 00:00:00+00:00
[2023-10-20 23:22:43,324] {standard_task_runner.py:52} INFO - Started process 2918 to run task
[2023-10-20 23:22:43,327] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2023-05-19T00:00:00+00:00', '--job-id', '884', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpbe4nvzwr', '--error-file', '/tmp/tmpuyfyo8yh']
[2023-10-20 23:22:43,328] {standard_task_runner.py:80} INFO - Job 884: Subtask segunda_tarea
[2023-10-20 23:22:43,416] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-05-19T00:00:00+00:00 [running]> on host dfdda228c9fc
[2023-10-20 23:22:43,536] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2023-05-19T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-19T00:00:00+00:00
[2023-10-20 23:22:43,538] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-10-20 23:22:43,539] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-10-20 23:22:43,549] {subprocess.py:85} INFO - Output:
[2023-10-20 23:22:43,551] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-10-20 23:22:43,552] {subprocess.py:96} INFO - Command exited with return code 0
[2023-10-20 23:22:43,590] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20230519T000000, start_date=20231020T232243, end_date=20231020T232243
[2023-10-20 23:22:43,620] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-20 23:22:43,665] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
